whittle/metrics/latency.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import torch.profiler
+from torch.profiler import record_function
+
+
+# Define a sample language model (e.g., a simple RNN or transformer-based model)
+def get_total_cpu_gpu_runtime(prof):
+    # from torch.autograd.profiler_utils.py (_build_table)
+    # results is in micro-seconds
+    from torch.autograd import DeviceType
```

Comment:
Is there a reason to have the import here instead on top of the file?

Reply:
Addressed